### ISSUE TITLE
A proper error message should be printed if rsvg-convert was not found

### ISSUE THREAD

**Issue by p-deli** [openzim/mwoffliner#491]
Just installed mwoffliner in Ubuntu 18.10 Desktop.
When issuing `mwoffliner --mwUrl=http://192.168.56.5/UnifiedICTDoc --adminEmail=foo@bar.net --verbose` or `mwoffliner --mwUrl=https://ts.wikipedia.org --adminEmail=foo@bar.net --verbose`, I get this:

```
Failed to run mwoffliner after [0s]: {
	"killed": false,
	"code": 127,
	"signal": null,
	"cmd": "rsvg-convert --version"
}
```
A proper error could perhaps be displayed…?

`sudo apt install librsvg2-bin` solved the problem… (although a new one appeared, but that is another issue I am investigating)

**Comment by kelson42** [comment]
@p-deli Agree

**Comment by ISNIT0** [comment]
For this matter, we could do with proper error messages for all of the required commands: https://github.com/openzim/mwoffliner/blob/master/src/Utils.ts#L102

**Comment by kelson42** [comment]
@ISNIT0 yes, we should never see unhandled error IMO.

**Comment by kelson42** [comment]
Regression: MWoffliner still crash (or we have a regression) if `convert` is not installed.

**Comment by kelson42** [comment]
If you could fixed this trivial one, that would be welcome.

**Comment by kelson42** [comment]
We don't use `rsvg-convert` anymore and `convert` won't be used anymore after #1069 is fixed.
